keycloak/keycloak-admin-ui
# Added the credentials tab

## Motivation
Added credentials tab [marvel](https://marvelapp.com/prototype/695h94c/screen/72265889)

## Verification Steps

1. Go to client details
1. Make the client "public"
1. The new credentials tab will appear
-->

## Checklist:


## Additional Notes
Setting the client to see the credentials tab:
![image](https://user-images.githubusercontent.com/51133/97539091-b6220f00-19c1-11eb-97d1-464ef3520ee9.png)
